client/app/vps/cloud-database/order/vps-cloud-database-order.html

```diff
@@ -0,0 +1,129 @@
+        <p data-translate="vps_tab_cloud_database_order_explanation_text"></p>
+    </div>
+    <h3 data-translate="vps_tab_cloud_database_order_customize"></h3>
+    <hr />
+    <form name="cloudDatabaseOrderForm">
+
+        <div>
+            <label class="required"
+                   for="orderVersion"
+                   data-translate="vps_tab_cloud_database_order_version"></label>
+            <oui-select
+                data-placeholder="{{:: 'common_select' | translate }}"
+                data-model="$ctrl.currentOrder.version"
+                data-items="$ctrl.versions"
+                data-match="name"
+                data-on-change="$ctrl.showOrRefreshDurations()"
+                data-required>
+                <span data-ng-bind="$item.name"></span>
+            </oui-select>
+        </div>
+
+        <div>
+            <label class="required"
+                   for="orderRam"
+                   data-translate="vps_tab_cloud_database_order_ram"></label>
+            <oui-select
+                data-placeholder="{{:: 'common_select' | translate }}"
+                data-model="$ctrl.currentOrder.ram"
+                data-items="$ctrl.ramAmounts"
+                data-match="name"
+                data-on-change="$ctrl.showOrRefreshDurations()"
+                data-required>
+                <span data-ng-bind="$item.name"></span>
+            </oui-select>
+        </div>
+
+        <div>
+            <label class="required"
+                   for="orderDatacenter"
+                   data-translate="vps_tab_cloud_database_order_datacenter"></label>
+            <oui-select
+                data-placeholder="{{:: 'common_select' | translate }}"
+                data-model="$ctrl.currentOrder.datacenter"
+                data-items="$ctrl.datacenters"
+                data-match="name"
+                data-on-change="$ctrl.showOrRefreshDurations()"
+                data-required>
+                <span data-ng-bind="$item.name"></span>
+            </oui-select>
+        </div>
+
+        <oui-spinner data-size="m" data-ng-if="$ctrl.loading.durations"></oui-spinner>
+
+        <div data-ng-if="!$ctrl.loading.durations && $ctrl.durations">
+
+            <h3 class="mt-5" data-translate="vps_tab_cloud_database_order_period_title"></h3>
+
+            <fieldset>
+                <legend class="border-0"
+                        data-translate="vps_tab_cloud_database_order_period_subtitle"></legend>
+                <oui-radio-group data-model="$ctrl.currentOrder.duration">
+                    <oui-radio data-ng-repeat="duration in $ctrl.durations"
+                               data-value="duration">
+                        <span data-ng-bind="duration.value | duration:'longDate'"></span>&nbsp;:
+                        <span data-ng-bind-html="duration.details.prices | price:$ctrl.user.ovhSubsidiary"></span>
+                        <oui-spinner data-size="s" data-ng-if="!duration.details.prices"></oui-spinner>
```

Comment:
I think you should display no data as long ad you don't have all the data

Reply:
CX prefer that way.